trieste/acquisition/_optimizer.py

```diff
@@ -62,11 +63,19 @@ def _box(space: Box, target_func: TensorMapping) -> QueryPoints:
     initial_point = optimize(trial_search_space, target_func)
 
     bijector = tfp.bijectors.Sigmoid(low=space.lower, high=space.upper)
-    variable = tf.Variable(bijector.inverse(initial_point))
 
-    def _objective() -> tf.Tensor:
-        return -target_func(bijector.forward(variable))
+    def _objective(x: tf.Variable) -> tf.Tensor:
+        return tf.squeeze(-target_func(bijector.forward(x)), axis=-1)
 
-    gpflow.optimizers.Scipy().minimize(_objective, (variable,))
+    lbfgs_result = tfp.optimizer.lbfgs_minimize(
+        lambda x: tfp.math.value_and_gradient(_objective, x),
+        tf.Variable(bijector.inverse(initial_point)),
```

Comment:
that this needs to be a `Variable` is a bug in tfp afaict, see https://github.com/tensorflow/probability/issues/1182